Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org/"
+
+gem 'rambling-trie-opal', :git => 'https://github.com/interscript/interscript-js.git'
```

Comment:
@webdev778 can you change this to just `gem 'rambling-trie-opal'`? I just published it.